.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -15,6 +15,17 @@ As a [type of user],
 I want [an action or feature],
 So that [benefit or reason].
 
+# Work impact
+<!--
+Filled in by the Product Owner or the Stakeholders.
+-->
+
+Answer the following questions:
+- Is there currently a workaround for this issue? If so, what is it?
+- How much time would be saved by implementing this feature on a weekly basis?
+- How many users are affected by this issue?
+
+
```

Comment:
I noticed that the impact on the customer may affect the priority. Is that something you want to consider here?

Reply:
Yes, good idea